utils/syscall.c

```diff
@@ -440,6 +773,30 @@ long myst_syscall_num(const char* name)
     return -ENOENT;
 }
 
+const int* myst_syscall_group(const char* name)
+{
+    for (size_t i = 0; _groups[i].name; i++)
```

Comment:
Would `i` goes out of bound? There is no NULL element at the end of  `_groups`

Reply:
Added the null element